tests/e2e/sijoittelun-tulokset.spec.ts

```diff
@@ -419,6 +419,119 @@ test.describe('accepting valintaesitys', () => {
   });
 });
 
+test.describe('other actions of hakemus', () => {
+  test('show changehistory', async ({ page }) => {
+    await page.route(
+      '*/**/valinta-tulos-service/auth/muutoshistoria*',
+      async (route) => {
+        await route.fulfill({
+          status: 200,
+          json: [
+            {
+              changes: [
+                {
+                  field: 'valinnantilanViimeisinMuutos',
+                  from: '2024-05-14T09:52:43.341+03:00',
+                  to: '2024-10-02T14:38:11.336+03:00',
+                },
+                {
+                  field: 'valinnantila',
+                  from: 'HYLATTY',
+                  to: 'PERUUNTUNUT',
+                },
+              ],
+              timestamp: '2024-10-02T14:36:00.955506+03:00',
+            },
+            {
+              changes: [
+                {
+                  field: 'valinnantilanViimeisinMuutos',
+                  to: '2024-05-14T09:52:43.341+03:00',
+                },
+                {
+                  field: 'valinnantila',
+                  to: 'HYLATTY',
+                },
+                {
+                  field: 'hyvaksyttyVarasijalta',
+                  to: false,
+                },
+                {
+                  field: 'hyvaksyPeruuntunut',
+                  to: false,
+                },
+                {
+                  field: 'julkaistavissa',
+                  to: false,
+                },
+              ],
+              timestamp: '2024-05-14T09:52:41.927195+03:00',
+            },
+          ],
+        });
+      },
+    );
+    await page
+      .locator(
+        '[id="other-actions-menu-1\\.2\\.246\\.562\\.11\\.00000000000001796027"]',
+      )
```

Comment:
Voisko tän tehdä näin? Simuloisi paremmin käyttäjän tekemistä ja olisi siten enemmän Playwrightin hyvien käytäntöjen mukaista. https://playwright.dev/docs/best-practices#best-practices
```suggestion
    await page.getByRole('row', {name: 'Nukettaja Ruhtinas' }).getByRole('button', {name: 'Muut toiminnot'})
```